lib/config.js

```diff
@@ -65,16 +65,16 @@ var ENV_TABLE = {
   maxQueueSize: 'ELASTIC_APM_MAX_QUEUE_SIZE',
   asyncHooks: 'ELASTIC_APM_ASYNC_HOOKS',
   ff_captureFrame: 'ELASTIC_APM_FF_CAPTURE_FRAME',
-  sourceContextErrorAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_APP_FRAMES',
-  sourceContextErrorLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES',
-  sourceContextSpanAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_APP_FRAMES',
-  sourceContextSpanLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES',
+  sourceLinesErrorAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_APP_FRAMES',
+  sourceLinesErrorLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES',
+  sourceLinesSpanAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_APP_FRAMES',
+  sourceLinesSpanLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES',
```

Comment:
s/CONTEXT/LINES/

(also on the 3 lines above)

Reply:
fixed